test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -1,15 +1,31 @@
 package seedu.addressbook.common;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import org.junit.Test;
 
 import java.util.Arrays;
-import java.util.List;
 
-import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNullTest() {
+        Utils testing = new Utils();
+
+        // Testing one null case
+        assertEquals(testing.isAnyNull((Object)null), true);
+
+        // Testing one object
+        assertEquals(testing.isAnyNull("Hello"), false);
+
+        // Testing more object with some null items
+        assertEquals(testing.isAnyNull((null),"Hello"), true);
+        assertEquals(testing.isAnyNull((null),"Hello","Bye"), true);
+        assertEquals(testing.isAnyNull((null), 1, 2), true);
+
```

Comment:
You could probably add a few more non null test cases. Since, you have tested only for a single non null object, you could probably pass a collection of non null objects and test again to return false. Better to test false and true cases with equal priority.